app/main/helpers/briefs.py

```diff
@@ -0,0 +1,72 @@
+import six
+
+from flask import abort, current_app, render_template
+from flask_login import current_user
+
+from dmutils.email import send_email, hash_email, MandrillException
+
+
+def get_brief(data_api_client, brief_id, live_only=True):
+    brief = data_api_client.get_brief(brief_id)['briefs']
+    if live_only and brief['status'] != 'live':
+        abort(404)
+
+    return brief
+
+
+def check_supplier_is_eligible_for_brief(brief, supplier_id):
```

Comment:
:warning: really nitpicky semantics :warning: 
Based on how this is used I think this would be better named something like `ensure_supplier_is_eligible_for_brief`. Check implies to me that the fact is being checked but not necessarily acted upon.